database/migrations/2023_07_03_115510_create_quote_progresses.php

```diff
@@ -15,6 +15,7 @@ public function up(): void
             $table->string('id')->primary();
             $table->string('email');
             $table->json('data');
+            $table->json('state');
```

Comment:
@DarkWasabi @DashaSyvyuk what is `state`?

Reply:
@mominsid it's form state for FE form hydration. I'd rather avoid that, but @laidav said he had issues. Please check this https://github.com/jauntin/gbli-events/issues/34#issuecomment-1634674399

Reply:
@DarkWasabi @laidav - ok got it. It might be worth it to step back and review some of the details, because I was also hoping to keep this as simple as possible.

But if this is the way to go, we might not need a validate endpoint, since the server has the data to validate against already, and it can provide validation information in it initial response.. That's just a thought, it may or may not be better.

Reply:
@DarkWasabi @DashaSyvyuk @mominsid , as per our meeting earlier I am going to upload the payload to following

We can save all of `data` as one column `$table->json('data')`;

<img width="434" alt="image" src="https://github.com/jauntin/saving-quote/assets/36419353/d7bcd64b-a882-4601-905d-4fd976fc98cf">